<?xml version='1.0' encoding='UTF-8'?>
<root version="2.3.8">
 <tabbed_widget name="Main Window" parent="main_window">
  <Tab containers="1" tab_name="tab2">
   <Container>
    <DockSplitter count="4" orientation="-" sizes="0.250564;0.249436;0.250564;0.249436">
     <DockArea name="...">
      <plot style="Lines" mode="TimeSeries">
       <range right="107.600672" bottom="-2.844569" top="5.445685" left="57.598721"/>
       <limitY/>
       <curve color="#1f77b4" name="/control_joint_lqr/accel_command/data"/>
       <curve color="#d62728" name="/control_joint_lqr/accel_command_test/data"/>
      </plot>
     </DockArea>
     <DockArea name="...">
      <plot style="Lines" mode="TimeSeries">
       <range right="107.600688" bottom="-6.100162" top="19.612199" left="57.598748"/>
       <limitY/>
       <curve color="#1ac938" name="/control_joint_lqr/lqr_command/data"/>
       <curve color="#ff7f0e" name="/control_joint_lqr/lqr_command_test/data"/>
      </plot>
     </DockArea>
     <DockArea name="...">
      <plot style="Lines" mode="TimeSeries">
       <range right="107.600105" bottom="-1.525000" top="62.525000" left="57.600651"/>
       <limitY/>
       <curve color="#f14cc1" name="/joystick/accelerator_cmd/data"/>
       <curve color="#9467bd" name="/joystick/accelerator_cmd_test/data"/>
      </plot>
     </DockArea>
     <DockArea name="...">
      <plot style="Lines" mode="TimeSeries">
       <range right="107.599757" bottom="19.201535" top="31.802544" left="57.598735"/>
       <limitY/>
       <curve color="#17becf" name="/raptor_dbw_interface/wheel_speed_report/front_left">
        <transform name="Scale/Offset" alias="/raptor_dbw_interface/wheel_speed_report/front_left[Outlier Removal]">
         <options value_scale="0.2778" time_offset="0" value_offset="0"/>
        </transform>
       </curve>
       <curve color="#bcbd22" name="/velocity_planner/des_vel_test/data"/>
       <curve color="#1f77b4" name="/control_joint_lqr/desired_velocity_mps/data"/>
      </plot>
     </DockArea>
    </DockSplitter>
   </Container>
  </Tab>
  <currentTabIndex index="0"/>
 </tabbed_widget>
 <use_relative_time_offset enabled="1"/>
 <!-- - - - - - - - - - - - - - - -->
 <!-- - - - - - - - - - - - - - - -->
 <Plugins>
  <plugin ID="DataLoad CSV">
   <default time_axis=""/>
  </plugin>
  <plugin ID="DataLoad ROS2 bags">
   <use_header_stamp value="false"/>
   <discard_large_arrays value="true"/>
   <max_array_size value="100"/>
  </plugin>
  <plugin ID="DataLoad ULog"/>
  <plugin ID="ROS2 Topic Subscriber">
   <use_header_stamp value="false"/>
   <discard_large_arrays value="false"/>
   <max_array_size value="true"/>
   <selected_topics>
    <topic name="/accel_control/diagnostic"/>
    <topic name="/accel_control/diagnostic_test"/>
    <topic name="/aw_localization/ekf/odom"/>
    <topic name="/aw_localization/ekf/odom_bestpos"/>
    <topic name="/aw_localization/ekf/top_bestpos"/>
    <topic name="/control/acc/des_acc"/>
    <topic name="/control_joint_lqr/accel_command"/>
    <topic name="/control_joint_lqr/accel_command_test"/>
    <topic name="/control_joint_lqr/lqr_command"/>
    <topic name="/control_joint_lqr/lqr_command_test"/>
    <topic name="/control_joint_lqr/lqr_error"/>
    <topic name="/control_joint_lqr/lqr_error_test"/>
    <topic name="/control_joint_lqr/track_idx_test"/>
    <topic name="/control_joint_lqr/valid_conditions_test"/>
    <topic name="/control_safety_layer/out/control_cmd"/>
    <topic name="/joystick/accelerator_cmd"/>
    <topic name="/joystick/accelerator_cmd/raw"/>
    <topic name="/joystick/accelerator_cmd_test"/>
    <topic name="/joystick/accelerator_cmd_test/raw"/>
    <topic name="/joystick/brake_cmd"/>
    <topic name="/joystick/brake_cmd/raw"/>
    <topic name="/joystick/brake_cmd/raw_test"/>
    <topic name="/joystick/brake_cmd_test"/>
    <topic name="/joystick/steering_cmd"/>
    <topic name="/localization/ekf/odom"/>
    <topic name="/novatel_bottom/imu/data"/>
    <topic name="/raptor_dbw_interface/pt_report"/>
    <topic name="/raptor_dbw_interface/wheel_speed_report"/>
    <topic name="/system/status"/>
    <topic name="/velocity_planner/des_vel_test"/>
    <topic name="/velocity_planner/diagnostic_test"/>
    <topic name="/control_joint_lqr/desired_velocity_mps"/>
    <topic name="/control_joint_lqr/desired_velocity_mps_test"/>
   </selected_topics>
  </plugin>
  <plugin ID="UDP Server"/>
  <plugin ID="WebSocket Server"/>
  <plugin ID="ZMQ Subscriber"/>
  <plugin status="idle" ID="CSV Exporter"/>
  <plugin status="idle" ID="ROS2 Topic Re-Publisher"/>
 </Plugins>
 <!-- - - - - - - - - - - - - - - -->
 <previouslyLoaded_Datafiles/>
 <previouslyLoaded_Streamer name="ROS2 Topic Subscriber"/>
 <!-- - - - - - - - - - - - - - - -->
 <customMathEquations/>
 <snippets/>
 <!-- - - - - - - - - - - - - - - -->
</root>

